docs/docs/.vuepress/config.js

```diff
@@ -215,14 +147,99 @@ module.exports = {
           "/subprocessors/",
         ],
       },
-      ["https://pipedream.com/support", "Support and Community"],
-      "/troubleshooting/",
+      // {
```

Comment:
You can just remove the commented-out section. We always have the old state in Git if we need to retrieve it.